src/plugins/embeddable/public/lib/containers/i_container.ts

```diff
@@ -28,6 +28,17 @@ export interface ContainerInput<PanelExplicitInput = {}> extends EmbeddableInput
   };
 }
 
+export interface EmbeddableContainerInitializeSettings {
+  /**
+   * If true, the container will wait for each embeddable to load after creation before loading the next embeddable.
+   */
+  awaitEachChild?: boolean;
```

Comment:
nit: I was thinking maybe the word "sequential" can be used here somehow, i.e:

```suggestion
  loadSequentially?: boolean;
```

Reply:
Sequentially is definitely a good word! I've named that `initializeSequentially`, so that it fits well in the broader `settings` object that @jloleysens suggested.